subiquity/controllers/filesystem.py

```diff
@@ -76,7 +78,30 @@ def __init__(self, app):
         self._crash_reports = {}
         self._probe_once_task = SingleInstanceTask(
             self._probe_once, propagate_errors=False)
-        self._probe_task = SingleInstanceTask(self._probe)
+        self._probe_task = SingleInstanceTask(
+            self._probe, propagate_errors=False)
+
+    def load_autoinstall_data(self, data):
+        log.debug("load_autoinstall_data %s", data)
+        if not self.interactive() and data is None:
+            data = {
```

Comment:
This is curious, what's going on here?

Reply:
This is trying to have a non-interactive only default, I think.